Source/REFACTORING/Collection/DataSource/DiffableCollectionDataSource.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  DiffableCollectionDataSource.swift
+//  ReactiveDataDisplayManager
+//
+//  Created by Никита Коробейников on 15.03.2021.
+//
+
+import UIKit
+
+/// DataSource based on `UICollectionViewDiffableDataSource` with automatic cells managing
+///
+/// - Warning: Required to conform all generators to `DiffableItemSource`
+@available(iOS 13.0, *)
+open class DiffableCollectionDataSource: UICollectionViewDiffableDataSource<DiffableItem, DiffableItem>, CollectionDataSource {
+
+    // MARK: - Properties
+
+    weak public var provider: CollectionGeneratorsProvider?
```

Comment:
Вроде везде так у нас было по кодстайлу

```suggestion
    public weak var provider: CollectionGeneratorsProvider?
```